debezium-platform-conductor/src/main/java/io/debezium/platform/environment/operator/configuration/TableNameResolver.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.platform.environment.operator.configuration;
+
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.regex.Pattern;
+
+import jakarta.enterprise.context.Dependent;
+
+import io.debezium.platform.domain.views.flat.PipelineFlat;
+
+@Dependent
+public class TableNameResolver {
+
+    private static final Map<String, Function<PipelineFlat, String>> PLACE_HOLDERS = Map.of("@{pipeline_name}", PipelineFlat::getName);
+
+    public String resolve(PipelineFlat pipeline, String currentValue) {
+
+        if (currentValue == null || currentValue.isEmpty()) {
+            return currentValue;
+        }
+
+        String processedValue = PLACE_HOLDERS.entrySet().stream()
+                .reduce(currentValue,
+                        applyReplacementFunction(pipeline),
+                        (lastProcessed, newProcessed) -> newProcessed);
+
+        return sanitizeTableName(processedValue);
+    }
+
+    private BiFunction<String, Map.Entry<String, Function<PipelineFlat, String>>, String> applyReplacementFunction(PipelineFlat pipeline) {
```

Comment:
What about we don't do that? If you must, create a wrapper class, however this is a bit too much I think :)

Reply:
What do you think is too much? The function itself?

Reply:
@jcechace I reworked it. It think it is better readable now but still flexible. WDYT?

Reply:
LGTM... yea, the data type was a bit insane ;)